.github/workflows/release.yml

```
@@ -22,11 +22,8 @@ jobs:
         with:
           token: ${{ secrets.GH_RELEASE_PAT }}
           fetch-depth: 0
-      - uses: getsentry/craft@master
-        name: Craft Prepare
-        with:
-          action: prepare
-          version: ${{ env.RELEASE_VERSION }}
+      - name: Craft Prepare
+        run: npx @sentry/craft prepare ${{ env.RELEASE_VERSION }}
```

I'm not entirely sure, but shouldn't this have `--no-input`? Also, to be safe I'd quote the release because it can contain `@`.

```suggestion
        run: npx @sentry/craft prepare --no-input "${{ env.RELEASE_VERSION }}"
```

---

This is not necessary since craft checks the `CI` env var. Also, escaping is likely not necessary for this repo.